motif-docs/pages/guides/advanced/deploy-prebuilt-image.mdx

```diff
@@ -0,0 +1,46 @@
+---
+heading: Beyond the basics
+title: Deploy a Prebuilt Container Image
+description: Using Image Registries to deploy a pre-built image
+noTOC: false
+---
+
+import Image from 'next/image'
+
+import imageRegistryGuiPic from '../../../public/img/image-registry-gui.png'
+import imageRegistryTablePic from '../../../public/img/image-registry-table.png'
+
+# Introduction
+
+Flightcontrol allows you to deploy your own pre-built image with an image registry, which allows you to use the platform as CD, instead of only CI/CD. To clarify our terminology, we use "registry" to refer to the host service that stores a collection of image repostorities, which we refer to as a "repository."
```

Comment:
```suggestion
Flightcontrol allows you to deploy your own pre-built container image with an image registry, which allows you to use the platform as CD, instead of only CI/CD. To clarify our terminology, we use "registry" to refer to the host service that stores a collection of image repostorities, which we refer to as a "repository."
```